lib/components/user/notification-prefs-pane.tsx

```diff
@@ -40,7 +42,7 @@ const NotificationOption = styled(ListGroupItem)`
     text-transform: uppercase;
   }
   label + span {
-    color: ${GRAY_ON_WHITE};
+    color: ${grey[700]};
```

Comment:
Is there any indication in the code, other than `700`, to tell that this particular shade is meant to be used for, in this instance, gray-on-white text, as opposed to, say `600` and the possibility that a11y tests could fail? I think it would make sense to define aliases for some of the most used indexes/colors, so that we use them consistently long-term.

Reply:
Really good point, reinstated the color variables in #1207!